openfisca_web_api/controllers/calculate.py

```diff
@@ -85,7 +85,7 @@ def calculate_simulations(scenarios, variables, trace):
                 except ParameterNotFound as exc:
                     raise wsgihelpers.respond_json(ctx,
                         collections.OrderedDict(sorted(dict(
-                            apiVersion = 1,
+                            apiVersion = environment.country_package_version,
```

Comment:
Not sure `apiVersion` is the clearest naming. `country_version` ? `legislation_version` ?